### DISCUSSION TITLE
MonitorControl v4.3.2 is now available with Sequoia compatibility!

### DISCUSSION THREAD

**Discussion by waydabber** [MonitorControl/MonitorControl#1633] [General]
This MonitorControl  version provides **macOS Sequoia** compatibility and various localization updates.

Please note that due to a change in signatures the v4.2.0 app version does not upgrade to this version - you need to download and install this version manually. You might need to configure Accessibility Permissions again for native keyboard control to work properly.

For more features and better compatibility with Sequoia consider switching to **[BetterDisplay](https://betterdisplay.pro)**.

## What's Changed

- Fixed: App crashes on macOS 15.0 Sequoia - @waydabber
- Fixed: Dock is frozen after quitting the app in Sequoia - @waydabber
- Fixed: Volume control not available when the display is in HDR mode in Sequoia - @waydabber
- Fixed: Custom keyboard shortcuts don't work in Sequoia - @waydabber
- Added the Portuguese translation - @eduardosantoshf
- Added Hindi translation - @patiljignesh
- Added Japanese translation - @shsw228- Updated Russian translation - @ghostiam
- Added Slovak translation - @Svec-Tomas
- Updated French translation - @botho
- Updated Spanish translation - @stonkol
- Updated Chinese (Taiwan) translation - @stonkol
- Updated Turkish translation - @Zaryob

<hr /><em>This discussion was created from the release <a href='https://github.com/MonitorControl/MonitorControl/releases/tag/'>MonitorControl v4.3.2</a>.</em>

**Comment by stevenya97** [comment]
Thanks for the update! Able to go into detail on what causes the signature to change on this one compared to prior updates?

**Reply by waydabber** [reply]
Yes, we had a change in the list of maintainers and the guy who previously signed the app is no longer an active member of the team. Due to technical reasons we could not make the transition any smoother.

I now essentially took over the project. Although I focus mainly on BetterDisplay, I plan to keep MonitorControl going as well for the sake of those who opt for a simpler, open source solution for brightness control.

**Reply by wclyffe4** [reply]
I can't thank you enough @waydabber 🌟

**Reply by stevenya97** [reply]
@waydabber Awesome thanks for the clarity! At least the situation seems reasonable enough for the end user. Was hoping it wasn't a similar situation like Bartender where the project was transferred over to a third party entity entirely. I appreciate your all important work on both MonitorControl and BetterDisplay
